tools/run_tests/run_tests.py

```diff
@@ -355,7 +355,7 @@ def test_specs(self):
                             tests = subprocess.check_output(
                                 [binary, '--benchmark_list_tests'],
                                 stderr=fnull)
-                        for line in tests.split('\n'):
+                        for line in tests.decode().split('\n'):
```

Comment:
should or should we not specify the encoding here explicitly?

Reply:
[The default is "UTF-8"](https://docs.python.org/3/library/stdtypes.html#bytes.decode). This should be fine.